series/supportedseries.go

```diff
@@ -316,17 +324,22 @@ var windowsNanoVersions = map[string]string{
 }
 
 // WindowsVersions returns all windows versions as a map
-func WindowsVersions() map[string]string {
+// If we have nan and windows version in common, nano takes precedence
+func WindowsVersions() (map[string]string, []string) {
```

Comment:
Is there a reason to change the signature of this, other than for tests? Maybe the test could have it's own function called `overwrittenWindowsVersions` that takes the `WindowsVersions` result and give you back what is missing.

```go
func overwrittenWindowsVersions(versions map[string]string) ([]string) {
	var overwrittenValues []string
	for i, val := range windowsNanoVersions {
		if overwritten, ok := versions[i]; ok {
			overwrittenValues = append(overwrittenValues, overwritten)
		}
	}
	return overwrittenValues
}
```